.env.example

```diff
@@ -54,6 +54,7 @@ PP_CACHED_ACCS=90,95,98,99,100
 DISALLOWED_NAMES=mrekk,vaxei,btmc,cookiezi
 DISALLOWED_PASSWORDS=password,abc123
 DISALLOW_OLD_CLIENTS=True
+DISALLOW_INGAME_REGISTRATION=False
```

Comment:
i might honestly argue that the default should be `True` despite the clear change in behaviour, it being enabled has caught out a lot of servers because of botting and you should probably be aware of the risks if you're going to use this

Reply:
yeah fair